### ISSUE TITLE
Big nothing burger

### ISSUE THREAD

**Issue by 23atomist** [rabbitscam/rabbitr1#7]
Anybody even look at the code. Big nothing burger. Bunch of junk with some whipped cream and doesn't even include a happy ending.

Don't waste your time on this BS.

**Comment by knoopx** [comment]
yeah spend 10 minutes checking the "source". nothing there at all.
